pallets/offchain-worker/src/lib.rs

```diff
@@ -101,9 +101,11 @@ pub mod pallet {
 	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
 		fn on_initialize(block_number: T::BlockNumber) -> Weight {
 			log::info!("ocw on_initialize {:?}.", block_number);
+// SBP M1: magic value?
 			1000
 		}
 
+// SBP M1: lacking weight? Might be simpler to push it in `on_initialize`
```

Comment:
on_finalize method not return a weight.

Reply:
Right it must be done in `on_initialize`

Reply:
understood, thanks